src/clients/ntp-client_proposal.rb

```diff
@@ -1,11 +1,18 @@
 require "yast"
 
+require "y2ntp_client/widgets/sources_table"
+
 module Yast
   # This is used as the general interface between yast2-country
   # (time,timezone) and yast2-ntp-client.
   class NtpClientProposalClient < Client
```

Comment:
Taking into account the big changes you are doing maybe it is time to move it to **lib/y2ntp_client** and use the **YaST2** Installation/ProposalClient and start testing it properly.

Reply:
sorry but the client is used in a way that trying to do it properly will be another major step which will also include redesigning yast2-country